### DISCUSSION TITLE
How to find support for Vercel?

### DISCUSSION THREAD

**Discussion by kriversdb** [vercel/community#6478] [Help]
### Summary

Going to the website if offers any chat support or phone support for this product. How do you find out if your company has an account since we are pointing our DNS to your service? 

### Steps to Reproduce

We went to the website and could not find a method to get assistance.

**Comment by amyegan** [comment]
Hey @kriversdb. If you go to [vercel.com/help](https://vercel.com/help), you can find options to get help from the Vercel community here, search for answers from our documentation and guides, or submit a case to our Customer Success Team. 

If you tell me more about the issue you're facing, I would be happy to assist.
